### DISCUSSION TITLE
Anything wrong with initializing all GetX Controllers @ start of main?

### DISCUSSION THREAD

**Discussion by BlackCatDev-IO** [jonataslaw/getx#1013] [Q&A]
Any reason not to do this and just use Get.find in the rest of your app, or is it better to initialize locally where you need it?

**Comment by jonataslaw** [comment]
You can have a startup DI container.
The difference is that your controllers will be in memory unnecessarily.
However you can use lazyPut + fenix mode to fix this.

**Reply by BlackCatDev-IO** [reply]
Understood, thanks for the fast response!
